lib/Runtime/Library/TypedArray.cpp

```diff
@@ -2844,7 +2844,7 @@ namespace Js
         return false;
     }
 
-    BOOL TypedArrayBase::ValidateIndexAndDirectSetItem(__in Js::Var index, __in Js::Var value, __in bool * isNumericIndex)
+    BOOL TypedArrayBase::ValidateIndexAndDirectSetItem(__in Js::Var index, __in Js::Var value, __out bool * isNumericIndex)
```

Comment:
You can use _Use_decl_annotations_ here so that you don't have to annotate both function declarations and definitions.